src/components/Results.vue

```diff
@@ -3,7 +3,7 @@
 
     <slot name="header"></slot>
 
-    <slot v-for="result in results" :result="result" :key="result.objectID">
+    <slot v-for="result in results" :result="result">
```

Comment:
Why was this removed? Now images will be replaced weirdly, right?

Reply:
The `slot` can't have the `:key` attribute (an error is raised) but the `template` can (see [this issue](https://github.com/vuejs/vue/issues/5575)).

Reply:
> Why was this removed? Now images will be replaced weirdly, right?

The key wasn't used so far. We should try to use a key on every hit to see if that fixes the images issue.